torch/onnx/_internal/exporter/_onnx_program.py

```diff
@@ -254,6 +255,20 @@ def release(self) -> None:
             self._tempdir.cleanup()
             self._tempdir = None
 
+    def _rename_dynamic_axes(
+        self,
+        dynamic_shapes: dict[str, Any] | tuple[Any, ...] | list[Any],
+    ) -> None:
+        """Rename dynamic axes in a model according to the specified dynamic_axes names."""
+        rename_mapping = _dynamic_shapes.create_rename_mapping(
+            self.model.graph.inputs, dynamic_shapes
+        )
+        _dynamic_shapes.iterate_and_change_axis_names(self.model, rename_mapping)
+        for model_function in self.model.functions.values():
+            _dynamic_shapes.iterate_and_change_axis_names(
+                model_function, rename_mapping
+            )
```

Comment:
```suggestion
        _dynamic_shapes.iterate_and_change_axis_names(self.model, rename_mapping)
```

model the function iteration into the iterate_and_change_axis_names pass

Reply:
Done